r/R/dplyr-funcs-datetime.R

```diff
@@ -15,6 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 
+check_locale <- function(locale = Sys.getlocale("LC_TIME")) {
```

Comment:
It looks like this is only about LC_TIME and its use in strftime, so let's name the function and revise the error message to reflect that